chart/kubeapps/templates/mongodb-secret-bootstrap.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: batch/v1
```

Comment:
we should rename the filenames to match the naming, i.e. mongodb-jobs-bootstrap here